src/Compilers/VisualBasic/Test/Emit/CodeGen/CodeGenTuples.vb

```diff
@@ -5937,6 +5937,178 @@ End Module
 
         End Sub
 
+        <Fact>
+        Public Sub CaseSensitivity001()
+
+            Dim verifier = CompileAndVerify(
+<compilation>
+    <file name="a.vb">
+Module Module1
+    Sub Main()
+        Dim x2 = (A:=10, B:=20)
+        System.Console.Write(x2.a)
+        System.Console.Write(x2.item2)
+        
+        Dim x3 = (item1 := 1, item2 := 2)
+        System.Console.Write(x3.Item1)
+        System.Console.WriteLine(x3.Item2)
+
+    End Sub
+End Module
+    </file>
+</compilation>, additionalRefs:=s_valueTupleRefs, expectedOutput:=<![CDATA[102012]]>)
+
+        End Sub
+
+        <Fact>
+        Public Sub CaseSensitivity002()
+
+            Dim comp = CreateCompilationWithMscorlibAndVBRuntime(
+<compilation>
+    <file name="a.vb">
+        <![CDATA[
+
+Module Module1
+    Sub Main()
+        Dim x1 = (A:=10, a:=20)
+        System.Console.Write(x1.a)
```

Comment:
(referring to the line
```vb
        Dim x1 = (A:=10, a:=20)
```

---
In reply to: [76714136](https://github.com/dotnet/roslyn/pull/13415#discussion_r76714136) [](ancestors = 76714136)